harness/determined/common/api/bindings.py

```diff
@@ -86,16 +86,26 @@ def __str__(self) -> str:
 
 
 class GetMasterResponseProduct(DetEnum):
+    """- PRODUCT_UNSPECIFIED: Not a Cloud Community offering
```

Comment:
this enum doesn't have a description?

Reply:
this was due to a missing piece in the parser for inlined enum definitions. I addressed it

![Screenshot 2023-09-13 at 17 59 17](https://github.com/determined-ai/determined/assets/12127420/52afccab-4f4e-418e-86a6-897398727345)